modules/33_CDR/DAC/equations.gms

```diff
@@ -35,14 +33,12 @@ q33_emicdrregi(t,regi)..
 	v33_emiDAC(t,regi);
 	
 ***---------------------------------------------------------------------------
-*'  Calculation of energy demand of direct air capture. Heat can be provided by gas or H2; 
-*'  vm_otherFEdemand(t,regi,"fegas") is calculated as the total energy demand for heat minus what is already covered by h2, i.e. vm_otherFEdemand(t,regi,"feh2s") and vice versa.
+*'  Calculation of energy demand of direct air capture. DAC demands feels and fehes
 ***---------------------------------------------------------------------------
 q33_otherFEdemand(t,regi,entyFe)..
     vm_otherFEdemand(t,regi,entyFe)
```

Comment:
Just general: ``vm_otherFEdemand`` could at some point be replaced by some form of ``vm_demFeSector(t,regi,entySe,entyFe,"cdr","ETS")``.